code/game/objects/items/weapons/null_rod.dm

```diff
@@ -140,15 +141,126 @@
 	icon_state = "cultblade"
 	item_state = "cultblade"
 
+/obj/item/weapon/nullrod/sword/armblade //Changeling religion
+	name = "exalted arm blade"
+	desc = "A holy blade made of flesh, bone and faith."
+	icon_state = "armblade"
+	item_state = "armblade"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	hitsound = "sound/weapons/bloodyslice.ogg"
+	cant_drop = TRUE
+
+/obj/item/weapon/nullrod/armblade/dropped()
+	var/obj/item/weapon/nullrod/N = new /obj/item/weapon/nullrod
+	qdel(src)
+	visible_message("<span class='notice'>\The [name] loses [ticker.Bible_deity_name]'s power and changes back into \a [N.name]</span>")
```

Comment:
Wouldn't it be better to have it go for the chaplains religious deity?

Reply:
Wait, what. oh, right. we have multiple religions now